src/main/tools/linux-sandbox-pid1.cc

```diff
@@ -349,8 +413,13 @@ static void SetupNetworking() {
 }
 
 static void EnterSandbox() {
```

Comment:
WDYT, should we rename this to EnterWorkingDirectory?

Reply:
I think it makes sense, done